build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -7,12 +7,8 @@ parameters:
 # The "primary" build arch is the one that the nuspec gets its winmd, pri, and other neutral files from
   primaryBuildArch: x86
```

Comment:
Is x86 the touchstone we want to use?
Wondering if x64 wouldn't make a better choice

Just a thought

Reply:
This parameter is not being used either